python/tests/test_read_conformance.py

```diff
@@ -0,0 +1,30 @@
+import difflib
+import unittest
+from pathlib import Path
+
+from mcap.mcap0.records import DataEnd
+from mcap.mcap0.serialization import stringify_record
+from mcap.mcap0.stream_reader import StreamReader
+
+
+class McapStreamingReaderConformanceTestCase(unittest.TestCase):
```

Comment:
does this still need to exist or can it be deleted since you added the test runner in TS?

Reply:
It's moderately useful since its much faster than the TS runner.